dhis-2/dhis-api/pom.xml

```diff
@@ -142,6 +142,10 @@
       <groupId>com.vividsolutions</groupId>
       <artifactId>jts-io</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+    </dependency>
```

Comment:
I am wondering if the use of lombok is a system-wide decision - perhaps better not to use this in just a few places for consistency reasons?

Reply:
Yes, I wanted to discuss this with you before reviewing. We briefly discussed the use of such frameworks during the interview. I agree that it doesn't make a lot of sense to use Lombok or similar tools in a "spotty" way. It was more a way to start the discussion.